decidim-core/app/packs/src/decidim/editor.js

```diff
@@ -74,13 +96,20 @@ export default function createQuillEditor(container) {
         next(file);
       }
     }
+
+    const text = $(container).data("dragAndDropHelpText");
+    $(container).after(`<p class="help-text" style="margin-top:-1.5rem;">${text}</p>`);
   }
   const quill = new Quill(container, {
     modules: modules,
     formats: quillFormats,
     theme: "snow"
   });
 
+  if (addImage === false) {
+    quill.root.addEventListener("drop", (ev) => ev.preventDefault());
```

Comment:
Maybe add a comment here explaining what this is for? Because this is a bit hard to test obviously as this works differently in Chrome.
```suggestion
    // Firefox natively implements image drop in contenteditable which is why we need to disable that
    quill.root.addEventListener("drop", (ev) => ev.preventDefault());
```